tags/1.4.9/includes/pardot-forms-shortcode-popup-class.php

```diff
@@ -42,7 +42,7 @@ public function load_js() {
 		wp_enqueue_script( 'pardot-popup-js', plugins_url( 'js/popup.js', PARDOT_PLUGIN_FILE ), array( 'pardot-chosen-js', 'jquery' ) );
 		wp_localize_script( 'pardot-popup-js', 'PardotShortcodePopup', array(
 			'ajaxurl'            => admin_url( 'admin-ajax.php' ),
-			'tinymce_button_url' => plugins_url( 'images/pardot-button.png', PARDOT_PLUGIN_FILE )
+			'tinymce_button_url' => plugins_url( 'images/salesforce-button.png', PARDOT_PLUGIN_FILE )
```

Comment:
@nsweeneypi  The ones under tag folder are historical version, should we just leave them? @adelawalla  fyi

Reply:
Ah, sorry, @pengchensfdc - I guess we cna keep them listed as 'Pardot'. Does that work, @adelawalla?

Reply:
@nsweeneypi , I talked to Adil in standup that we leave these historical version behind, you could revert these changes under tags folder.